lib/aliases.js

```diff
@@ -4,7 +4,13 @@ const aliases = {
   deb: ['debian'],
   rpm: ['fedora'],
   AppImage: ['appimage'],
-  dmg: ['dmg']
+  dmg: ['dmg'],
+  darwin_arm64: ['mac_arm64', 'macos_arm64', 'osx_arm64'],
+  exe_arm64: ['win32_arm64', 'windows_arm64', 'win_arm64'],
+  deb_arm64: ['debian_arm64'],
+  rpm_arm64: ['fedora_arm64'],
+  AppImage_arm64: ['appimage_arm64'],
+  dmg_arm64: ['dmg_arm64']
```

Comment:
Instead of repeating all of these, can we instead write some code that auto-adds the suffix to all of the entries? Like that, we only have a single list of aliases to maintain.